frontend/public/actions/dashboards.ts

```diff
@@ -138,7 +131,6 @@ export type WatchPrometheusQueryAction = (
   namespace?: string,
   timespan?: number,
 ) => ThunkAction;
-export type WatchAlertsAction = () => ThunkAction;
 export type StopWatchURLAction = (url: string) => void;
 export type StopWatchPrometheusAction = (query: string, timespan?: number) => void;
 export type StopWatchAlertsAction = () => void;
```

Comment:
Should this be removed like `WatchAlertsAction` has been?

Reply:
good catch! Removed